package.json

```diff
@@ -20,7 +21,7 @@
     "build:control": "rollup -c rollup.config.mjs",
     "build:flags": "RETAIN_FLAGS=true ember build --env production --suppress-sizes",
     "lint": "npm-run-all lint:*",
-    "lint:files": "dotenv -- turbo test:lint",
+    "lint:files": "dotenv -- turbo lint",
```

Comment:
lint now depends on publint, which also depends on build.
in another PR, I'll be adding an in-CI turbo server to help with build times